test/unit/src/server/handlers/t_login.js

```diff
@@ -1,9 +1,15 @@
 import expect from 'expect';
```

Comment:
It might be helpful to use whitespace-insensitive diffs from here on. https://github.com/canonical-ols/build.snapcraft.io/pull/95/files?w=1